compendium/modules/w05-classes-exercise.tex

```diff
@@ -89,7 +89,7 @@
 ~\\ \emph{fel} & \emph{typ} & \emph{förklaring} \\\hline
 
 \code|value is not a member of object|
-& kompileringsfel & det finns ingen instans med namnet \code|Punkt| \TODO{Förklara mer specifikt att detta fel ges då konstruktionsombudet ej innehåller variabeln x?}\\ %TODO
+& kompileringsfel & det finns ingen instans med namnet \code|Punkt|. (Det lite udda felmeddelandet syftar på att det i klassens autogenererade konstruktionsombud saknas en variabel med namnet \code|x|.)\\ % Footnote verkar inte funka, så förklaringen får tyvärr stå i själva tabellen.
```

Comment:
Remove the odd part; "Det lite udda felmeddelandet..." -> "Felmeddelandet ..."  Add `\Eng{constructor proxy}  and I'm still not sure about the Swedish translation, perhaps "konstruktörsombud" or "konstruktorombud" or "konstruktorproxy" are better alternatives. Please add an issue on that we need to investigate good translations for this. We create language praxis, so better be careful :)

Reply:
I've created an issue regarding the Swedish translations (#552). Is this explanation better?
```suggestion
& kompileringsfel & det finns ingen instans med namnet \code|Punkt|. (Felmeddelandet syftar på att det i klassens autogenererade konstruktorombud \Eng{constructor proxy} saknas en variabel med namnet \code|x|.)\\ % Footnote verkar inte funka, så förklaringen får tyvärr stå i själva tabellen.
```